Client/Application/AppDelegate.swift

```diff
@@ -199,6 +199,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         // Override point for customization after application launch.
         var shouldPerformAdditionalDelegateHandling = true
 
+        Profiler.setup() // NimbleDroid lib setup, runs in non-release only
```

Comment:
Is it not better to wrap this in a `#if NIMBLEDROID`?

Reply:
We should decide on a convention, either wrapping `Profiler` calls in `#if`, or keep the conditional contained in `Profiler`. I prefer the latter.